DESCRIPTION

```diff
@@ -48,6 +46,8 @@ Suggests:
     flexdashboard,
     ggplot2,
     httpuv,
+    httr,
+    jsonlite,
```

Comment:
Run up against the 20 packages in Imports again 😞 so I moved these to Suggests and them use them conditionally. These are mostly used only when making a _prediction_ from R so not a terrible option to move to Suggests.